runner.py

```diff
@@ -54,6 +55,14 @@ def run_agent(terminate_event, callback_event, config_file, name, team, index, m
 
 
 def main():
+    # Inject DLL
```

Comment:
can we actually move it to its own method that then gets called from main?

That way we can have more logic around if we want to perform dll injection at all